src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -559,9 +552,9 @@ public sealed partial class Utf8JsonWriter : System.IAsyncDisposable, System.IDi
         public void WritePropertyName(System.ReadOnlySpan<char> propertyName) { }
         public void WritePropertyName(string propertyName) { }
         public void WritePropertyName(System.Text.Json.JsonEncodedText propertyName) { }
-        public void WriteRawValue([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("Json")] string json, bool skipInputValidation = false) { }
         public void WriteRawValue(System.ReadOnlySpan<byte> utf8Json, bool skipInputValidation = false) { }
         public void WriteRawValue([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("Json")] System.ReadOnlySpan<char> json, bool skipInputValidation = false) { }
+        public void WriteRawValue([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("Json")] string json, bool skipInputValidation = false) { }
```

Comment:
Apparently this is a public API that was missed in the reference assembly.

Reply:
should we do this regen in seperate PR though so at least we can track quickly what changed?

Reply:
I don't think it's worth the added hassle. I've reviewed the changes manually and will mark the API changes introduced by this PR's implementation using comments.